### PR TITLE
ci: fix website main-page in pr

### PR DESCRIPTION
**Pull request by kayman233**
Починили отправку в s3 билда главной страницы веб-сайта для пулл-реквестов, из-за которой она не отображалась. Содержимое билда должно было находиться на уровень выше
Было - стало:
Снимок экрана 2023-09-14 в 13 01 20

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.700.6183499629.ts'>borderRadius_default_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.700.6183499629.ts'>borderRadius_plasma_b2c_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.700.6183499629.ts'>borderRadius_plasma_web_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.700.6183499629.ts'>borderRadius_sberHealth_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.700.6183499629.ts'>borderRadius_sbermarket_business_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.700.6183499629.ts'>borderRadius_sbermarket_metro_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.700.6183499629.ts'>borderRadius_sbermarket_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.700.6183499629.ts'>borderRadius_sbermarket_selgros_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.700.6183499629.ts'>borderRadius_sberonline_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.700.6183499629.ts'>borderRadius_sberprime_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.700.6183499629.ts'>borderRadius_stylesSalute_react-native--canary.700.6183499629.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.700.6183499629.swift'>color_default_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.700.6183499629.kt'>color_default_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.700.6183499629.ts'>color_default_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.700.6183499629.xml'>color_default_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.700.6183499629.swift'>color_plasma_b2c_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.700.6183499629.kt'>color_plasma_b2c_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.700.6183499629.ts'>color_plasma_b2c_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.700.6183499629.xml'>color_plasma_b2c_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.700.6183499629.swift'>color_plasma_web_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.700.6183499629.kt'>color_plasma_web_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.700.6183499629.ts'>color_plasma_web_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.700.6183499629.xml'>color_plasma_web_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.700.6183499629.swift'>color_sberHealth_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.700.6183499629.kt'>color_sberHealth_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.700.6183499629.ts'>color_sberHealth_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.700.6183499629.xml'>color_sberHealth_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.700.6183499629.swift'>color_sbermarket_business_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.700.6183499629.kt'>color_sbermarket_business_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.700.6183499629.ts'>color_sbermarket_business_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.700.6183499629.xml'>color_sbermarket_business_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.700.6183499629.swift'>color_sbermarket_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.700.6183499629.kt'>color_sbermarket_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.700.6183499629.swift'>color_sbermarket_metro_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.700.6183499629.kt'>color_sbermarket_metro_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.700.6183499629.ts'>color_sbermarket_metro_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.700.6183499629.xml'>color_sbermarket_metro_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.700.6183499629.ts'>color_sbermarket_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.700.6183499629.swift'>color_sbermarket_selgros_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.700.6183499629.kt'>color_sbermarket_selgros_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.700.6183499629.ts'>color_sbermarket_selgros_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.700.6183499629.xml'>color_sbermarket_selgros_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.700.6183499629.swift'>color_sbermarket_wlbusiness_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.700.6183499629.kt'>color_sbermarket_wlbusiness_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts'>color_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.700.6183499629.xml'>color_sbermarket_wlbusiness_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.700.6183499629.xml'>color_sbermarket_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.700.6183499629.swift'>color_sberonline_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.700.6183499629.kt'>color_sberonline_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.700.6183499629.ts'>color_sberonline_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.700.6183499629.xml'>color_sberonline_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.700.6183499629.swift'>color_sberprime_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.700.6183499629.kt'>color_sberprime_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.700.6183499629.ts'>color_sberprime_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.700.6183499629.xml'>color_sberprime_xml--canary.700.6183499629.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.700.6183499629.swift'>color_stylesSalute_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.700.6183499629.kt'>color_stylesSalute_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.700.6183499629.ts'>color_stylesSalute_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.700.6183499629.xml'>color_stylesSalute_xml--canary.700.6183499629.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.700.6183499629.ts'>shadow_default_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.700.6183499629.ts'>shadow_plasma_b2c_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.700.6183499629.ts'>shadow_plasma_web_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.700.6183499629.ts'>shadow_sberHealth_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.700.6183499629.ts'>shadow_sbermarket_business_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.700.6183499629.ts'>shadow_sbermarket_metro_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.700.6183499629.ts'>shadow_sbermarket_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.700.6183499629.ts'>shadow_sbermarket_selgros_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts'>shadow_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.700.6183499629.ts'>shadow_sberonline_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.700.6183499629.ts'>shadow_sberprime_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.700.6183499629.ts'>shadow_stylesSalute_react-native--canary.700.6183499629.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.700.6183499629.ts'>spacing_default_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.700.6183499629.ts'>spacing_plasma_b2c_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.700.6183499629.ts'>spacing_plasma_web_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.700.6183499629.ts'>spacing_sberHealth_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.700.6183499629.ts'>spacing_sbermarket_business_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.700.6183499629.ts'>spacing_sbermarket_metro_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.700.6183499629.ts'>spacing_sbermarket_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.700.6183499629.ts'>spacing_sbermarket_selgros_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts'>spacing_sbermarket_wlbusiness_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.700.6183499629.ts'>spacing_sberonline_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.700.6183499629.ts'>spacing_sberprime_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.700.6183499629.ts'>spacing_stylesSalute_react-native--canary.700.6183499629.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.700.6183499629.swift'>typo_mage_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.700.6183499629.kt'>typo_mage_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.700.6183499629.ts'>typo_mage_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.700.6183499629.swift'>typo_plasma_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.700.6183499629.kt'>typo_plasma_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.700.6183499629.ts'>typo_plasma_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.700.6183499629.swift'>typo_ruler_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.700.6183499629.kt'>typo_ruler_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.700.6183499629.ts'>typo_ruler_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.700.6183499629.swift'>typo_sage_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.700.6183499629.kt'>typo_sage_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.700.6183499629.ts'>typo_sage_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.700.6183499629.swift'>typo_sbermarket_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.700.6183499629.kt'>typo_sbermarket_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.700.6183499629.ts'>typo_sbermarket_react-native--canary.700.6183499629.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.700.6183499629.swift'>typo_soulmate_ios-swift--canary.700.6183499629.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.700.6183499629.kt'>typo_soulmate_kotlin--canary.700.6183499629.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.700.6183499629.ts'>typo_soulmate_react-native--canary.700.6183499629.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-700/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-700/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-700/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-700/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-700/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-700/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-700/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-700/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-700/

File: .github/workflows/documentation-pr.yml [hunk 1]
```diff
@@ -94,7 +94,7 @@ jobs:
         run: |
           export NODE_OPTIONS=--openssl-legacy-provider
           npm run build --prefix="./website/plasma-website"
-          cp -R ./website/plasma-website/build/ ./s3_build/${PR_NAME}
+          cp -R ./website/plasma-website/build/. ./s3_build/${PR_NAME}
```
Comment (Yakutoc) [inline .github/workflows/documentation-pr.yml:97]: @kayman233 А расскажи почему и что сделал? )

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-700/

File: .github/workflows/documentation-pr.yml (hunk 1 above)
Comment (kayman233) [reply .github/workflows/documentation-pr.yml:97]: описал в шапке

**Review by Yakutoc — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Pull request merged**